docs/tutorial/kymotracking.rst

```diff
@@ -387,3 +387,100 @@ By default, this will use the optimal number of lags (which in this case seems t
 .. image:: msdplot_100_lags.png
 
 It's not hard to see from this graph why taking too many lags results in unacceptably large variances.
+
+
+Dwelltime analysis
+------------------------
+
+The lifetime of the bound state(s) can be determined using `KymoLineGroup.fit_binding_times()`. This method defines
+the bound dwelltime as the length of each tracked line (in seconds) and optimizes the parameters of an
+exponential mixture distribution on these observations by Maximum Likelihood Estimation (MLE)
+:cite:`kaur2019dwell,woody2016memlet`.
```

Comment:
Could be simplified, eg: The lifetime of the bound state(s) can be determined using `KymoLineGroup.fit_binding_times()`. This method defines
the bound dwelltime as the length of each tracked line (in seconds). The lifetimes are then determined  using Maximum Likelihood Estimation (MLE) :cite:`kaur2019dwell,woody2016memlet`.  The user can choose the number of exponentials (components) that should to be used for the fit.